docs/standard/contributing.rst

```diff
@@ -64,6 +64,38 @@ For worked examples, see the `process note <https://docs.google.com/document/d/1
 
 #. Once approved by James, you can merge it yourself.
 
+Committing changes
+------------------
+
+Make atomic changes in one commit, rather than over many commits. For example, adding a definition to the schema and to the schema reference documentation. That way, reverting a commit doesn't leave the standard in an incoherent state.
```

Comment:
```suggestion
Make atomic changes in one commit, rather than over many commits: for example, when adding a definition to the schema, add it to the schema reference documentation in the same commit. That way, reverting a commit doesn't leave the standard in an incoherent state.
```